controller_manager/src/ros2_control_node.cpp

```diff
@@ -57,6 +57,12 @@ int main(int argc, char ** argv)
   auto cm = std::make_shared<controller_manager::ControllerManager>(
     executor, manager_node_name, "", cm_node_options);
 
+  std::string message;
+  if (!realtime_tools::lock_memory(message))
+  {
+    RCLCPP_WARN(cm->get_logger(), "Unable to lock the memory : '%s'", message.c_str());
+  }
+
```

Comment:
My impression is that this functionality isn't tied to the existence of an RT patch, correct? I'm just wondering if we could soften or put a condition on the warning and not spam the majority of people with a false alarm.

Reply:
You are right! It is not related to the RT patch. What conditioning can we add here to avoid spamming? I can't think of one right now

Reply:
I got 2 options here: 

* We can make it optional by a parameter `lock_memory` and by default we can make it false. If it is enabled, we have this executed and log being printed
* We can direct the output to `std::cerr` instead?

What do you prefer?

Reply:
I added a version with lock_memory parameter. Check it out and let me know what you think

Reply:
looks good, thank you!